dev/tasks/tasks.yml

```diff
@@ -1252,6 +1252,12 @@ tasks:
     ci: github
     template: r/github.linux.offline.build.yml
 
+  test-r-minsizerel:
+    ci: github
+    template: r/github.macos-linux.local.yml
```

Comment:
This builds on macos + linux, which isn't strictly necessary for triggering what we're trying to trigger, but might be worth checking both linux + macos anyway